### PR TITLE
feat(SnapCard, DataCard, DisplayDataCard): add aspect ratio support

### PR DESCRIPTION
**Pull request by marcoskolodny**
- Add aspect ratio in snap/data/displayData cards
- Change height to minHeight in snap and data cards, to avoid weird behaviour when content doesn't fit in the container.

### CODE REVIEW

File: src/__screenshot_tests__/__image_snapshots__/data-card-screenshot-test-tsx-data-card-with-aspect-ratio-1-1-1-snap.png
Comment (yceballost) [inline src/__screenshot_tests__/__image_snapshots__/data-card-screenshot-test-tsx-data-card-with-aspect-ratio-1-1-1-snap.png:1]: Just a comment, fucking weird the aspect ratio in data card with the current flex distribution xD but it is correct and expected by specs.

**Review by yceballost — APPROVED** [review]
perfect ✨

File: src/__screenshot_tests__/__image_snapshots__/data-card-screenshot-test-tsx-data-card-with-aspect-ratio-1-1-1-snap.png
Comment (yceballost) [reply src/__screenshot_tests__/__image_snapshots__/data-card-screenshot-test-tsx-data-card-with-aspect-ratio-1-1-1-snap.png:1]: Well this example is not bad
image

File: src/__screenshot_tests__/data-card-screenshot-test.tsx [hunk 1]
```diff
@@ -93,3 +94,16 @@ test('DataCard without icon, with top actions and too long title', async () => {
 
     expect(image).toMatchImageSnapshot();
 });
+
+test.each(ASPECT_RATIO_VALUES)('DataCard with aspect ratio %s', async (aspectRatio) => {
+    await openStoryPage({
+        id: 'components-cards-data-card--default',
+        args: {aspectRatio: aspectRatio.replace(':', ' ')},
+    });
```
Comment (atabel) [inline src/__screenshot_tests__/data-card-screenshot-test.tsx:102]: I'd use mobile viewport in this test. The resulting screenshots would be les weird

File: src/__stories__/data-card-story.tsx [hunk 2]
```diff
@@ -133,6 +144,18 @@ Default.argTypes = {
         options: ['button', 'link', 'button and link', 'on press', 'none'],
         control: {type: 'select'},
     },
+    aspectRatio: {
+        options: ['auto', '1 1', '16 9', '7 10', '9 10'],
```
Comment (atabel) [inline src/__stories__/data-card-story.tsx:148]:
```suggestion
        options: ['auto', ...fixedAspectRatioValues],
```

**Review by atabel — APPROVED** [review]

File: src/__screenshot_tests__/data-card-screenshot-test.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/__screenshot_tests__/data-card-screenshot-test.tsx:103]: done

**Review by atabel — APPROVED** [review]

**Pull request merged**